aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -10,6 +10,7 @@
 #include <ATen/native/cpu/zmath.h>
 #include <ATen/Parallel.h>
 
+#include <torch/csrc/autograd/functions/utils.h>
```

Comment:
Alban says we shouldn't do that: https://github.com/pytorch/pytorch/pull/57188#discussion_r626053784.

Reply:
Thanks for the pointer!